distribution/packages/src/common/scripts/preinst

```diff
@@ -27,6 +27,7 @@ case "$1" in
             adduser --quiet \
                     --system \
                     --no-create-home \
+                    --home /usr/share/elasticsearch \
```

Comment:
Why `/usr/share/elasticsearch`? Is `/var/lib/elasticsearch` more appropriate?

Reply:
I chose `/usr/share/elasticsearch` because it is ES_HOME, and it is not expected to be written to. I'm open to being persuaded either way.